includes/Fragments/TemplateOutput.php

```diff
@@ -44,6 +44,8 @@ final protected function setUpSmarty()
     {
         $this->smarty = new Smarty();
         $this->smarty->addPluginsDir($this->getSiteConfiguration()->getFilePath() . '/smarty-plugins');
+        $this->registerPlugin('modifier', 'count', 'count');
+        $this->registerPlugin('modifier', 'implode', 'implode');
```

Comment:
Is this supposed to be `$this->smarty->registerPlugin(...)`?

```suggestion
        $this->smarty->registerPlugin('modifier', 'implode', 'implode');
```

Looks like `count` [is supported](https://smarty-php.github.io/smarty/stable/designers/language-modifiers/language-modifier-count/) in Smarty 5.x ([and 4.5.4](https://github.com/smarty-php/smarty/issues/813#issuecomment-2270648968)), so we probably don't need to worry about that one

Reply:
> Is this supposed to be `$this->smarty->registerPlugin(...)`?
Yes, yes it is, I'll re-commit that in a moment

> Looks like `count` [is supported](https://smarty-php.github.io/smarty/stable/designers/language-modifiers/language-modifier-count/) in Smarty 5.x ([and 4.5.4](https://github.com/smarty-php/smarty/issues/813#issuecomment-2270648968)), so we probably don't need to worry about that one

Not according the to deprecation notice that Michael posted in the other PR, and doing it now will save hassle later?

Reply:
Yeah, we're currently on 4.5.3, and the count deprecation notice was seemingly removed in 4.5.4 as it's supported directly on 5.x

Reply:
> Yeah, we're currently on 4.5.3, and the count deprecation notice was seemingly removed in 4.5.4 as it's supported directly on 5.x

*sigh*